server/routes/index.ts

```diff
@@ -27,11 +28,16 @@ export default function routes(services: Services): Router {
     services.todayCache,
     services.contentfulService,
   )
-
+  const establishmentRollController = new EstablishmentRollController(services.establishmentRollService)
   const apiController = new ApiController(services.homepageService)
 
   get('/', getFrontendComponents(services, config.apis.frontendComponents.latest), homepageController.displayHomepage())
   post('/search', homepageController.search())
+  get(
+    '/establishment-roll',
+    getFrontendComponents(services, config.apis.frontendComponents.latest),
```

Comment:
Will all get routes want the `getFrontendComponents`? Could we add that to the `get` function above? Looks like in the profile we've done it like `get('/prisoner/*', getFrontendComponents(services, config.apis.frontendComponents.latest))`

Reply:
I know. This was done by @davidmiddletonmoj at the time, he did have a reason but I can't remember it. I don't think it is the job of this PR to sort that